content/en/docs/Responses/TranscodingProfile.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "TranscodingProfile"
+linkTitle: "TranscodingProfile [OS]"
+opensubsonic:
+- Extension
+description: >
+  Defines a transcoding profile.
+---
+
+{{< tabpane persist=false >}}
+{{< tab header="**Example**:" disabled=true />}}
+{{< tab header="OpenSubsonic" lang="json">}}
+{
+  "container": "mp3",
+  "audioCodec": "mp3",
+  "protocol": "http",
+  "maxAudioChannels": 2
+}
+{{< /tab >}}
+{{< /tabpane >}}
+
+| Field | Type | Req. | OpenS. | Details |
+| --- | --- | --- | --- | --- |
+| `container` | string | **Yes** | **Yes** | The container format (e.g., mp3, flac). |
+| `audioCodec` | string | **Yes** | **Yes** | The target audio codec for transcoding. |
```

Comment:
Probably worthwhile being a bit more explicit here (`audio/*`, or just `codec`)?

Reply:
all the values are started with audioXXXX to anticipate an eventual support for music videos at some point.
It's codec like mp3,aac, ...